docs/understanding-the-magic.rst

```diff
@@ -52,6 +52,16 @@ This allows you to access the session within the context of an incoming
 HTTP request, but it will *not* allow you access it outside that
 context.
 
+State & Security
+================
+
+One of the key features of :attr:`~OAuth2ConsumerBlueprint.session` is that 
```

Comment:
Please remove the `~`, so that the rendered HTML shows `OAuth2ConsumerBlueprint.session` instead of just `session`.